config/cola.conf

```diff
@@ -0,0 +1,23 @@
+include "defaults.conf"
```

Comment:
This doesn't look like it should be part of this commit. Is it here intentionally?

Reply:
Oh I actually I added the cola.conf file by mistake. I'll remove it from the commit.